src/Synergism.ts

```diff
@@ -631,7 +631,28 @@ export const player: Player = {
         'Loadout 7',
         'Loadout 8'
     ],
-    corruptionShowStats: true,
+    smartLoadouts: {  //If you add loadouts don't forget to add loadout names!
+        1: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        2: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        3: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        4: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        5: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        6: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        7: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        8: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    },
+    smartLoadoutNames: [
+        'c14', //No idea how to call them
+        'r8x25', //It wont self update names, if to change them
+        'w5x10',
+        'p2x1',
+        'p3x2',
+        'Omega',
+        'p4x2',
+        'Challenges'
+    ],
```

Comment:
Remove the loadout names and instead have it be the object keys of smartLoadouts

Reply:
I dont fully understand, I tryed this (I removed player.smartLoadoutNames from everywhere)
![image](https://user-images.githubusercontent.com/107661324/178784739-9367cab7-9f5a-4117-b761-a694f67d3467.png)